### PR TITLE
chore(docs): README refactor

### PR DESCRIPTION
**Pull request by bee344**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
 
-The purpose of these instances is to allow anyone to check and get a quick overview of the info that the asset-related endpoints provide.
+## Foreign Assets in Polkadot Asset Hub
 
-**NOTE: This instance should only be used for ad-hoc checks or tests and not for production, heavy testing or any other critical purpose.**
+Foreign assets are those assets in Asset Hub whose native blockchain is not Asset Hub. These are mainly native tokens from other parachains or bridged tokens from other consensus systems (such as Ethereum). Once a foreign asset has been registered in Asset Hub (by its root origin), users are enabled to send this token from its native blockchain to Asset Hub and operate with it as if it was any other asset.
```
Comment (IkerAlus) [inline README.md:95]: Add a paragraph above like:

"Assets, both native and foreign, are stored as a map from an ID to information about the asset, including a management team, total supply, total number of accounts, its sufficiency for account existence, and more. Additionally, the asset owner can register metadata like the name, symbol, and number of decimals for representation.

Some assets, as determined by on-chain governance, are regarded as “sufficient”. Sufficiency means that the asset balance is enough to create the account on-chain, with no need for the DOT existential deposit. Likewise, you cannot send a non-sufficient asset to an account that does not exist. Sufficient assets can be used to pay transaction fees (i.e. there is no need to hold DOT on the account). Assets do have a minimum balance, and if an account drops below that balance, the dust is lost. 

For an more details on general asset management in Asset Hub, refer to [the assets documentation](https://wiki.polkadot.network/docs/build-integrate-assets#assets-basics) of the Polkadot Wiki."

File: README.md [hunk 2]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
 
-#### Tx Wrapper Polkadot
+Practically speaking, foreign assets are handled by the `foreign-assets` pallet in Asset Hub, which is an instance of the previously mentioned `assets` pallet. Hence, this pallet exposes the same interface to users and other pallets as `assets` pallet. For example, to transfer a certain amount of a foreign asset (id) to an account (target), this pallet exposes the following call:
```
Comment (IkerAlus) [inline README.md]:
```suggestion
Practically speaking, foreign assets are handled by the `foreign-assets` pallet in Asset Hub, which is an instance of [the `assets` pallet](https://marketplace.substrate.io/pallets/pallet-assets/). Hence, this pallet exposes the same interface to users and other pallets as `assets` pallet. For example, to transfer a certain amount of a foreign asset (id) to an account (target), this pallet exposes the following call: 
```

File: README.md [hunk 3]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
-constructing parachain transactions, you can use `txwrapper-polkadot` exactly as on the Relay Chain, but construct transactions with the appropriate parachain metadata like  genesis hash, spec version, and type registry.
+``` ts
+foreignAssets.transferKeepAlive(id, target, amount)
+```
+
+The main difference to take into account for foreign assets is their identifier. Unlike using integers as identifiers in the `assets` pallet, assets stored in the `foreign-assets` pallet are identified by [their XCM multilocation](https://wiki.polkadot.network/docs/learn/xcm/fundamentals/multilocation-summary). Taking the example below, the id input parameter of the call above will be a multilocation type as `{parents: 1, interior: {X1: {Parachain: 2,123}}}`.
+
+### Foreign Asset Operations
+
+The main functions you will probably interact with are `foreignAssets.transfer_allow_death` and `foreignAssets.transfer_keep_alive`. These functions transfer some `amount` (balance) of an `AssetId` (a `u32`, not a contract address) to another account.
```
Comment (IkerAlus) [inline README.md]:
```suggestion
The main functions you will probably interact with are `foreignAssets.transfer_allow_death` and `foreignAssets.transfer_keep_alive`.

File: README.md [hunk 4]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
+
+The `foreign-assets` also provides an `approve_transfer`, `cancel_approval`, and `transfer_approved` interface for non-custodial operations.
+
+Note that you can use the same addresses (except [pure proxies](https://wiki.polkadot.network/docs/learn-proxies-pure#anonymous-proxy-pure-proxy)!) on the Asset Hub that
```
Comment (IkerAlus) [inline README.md]:
```suggestion
Note that you can use the same addresses (except [pure proxies](https://wiki.polkadot.network/docs/learn-proxies-pure#anonymous-proxy-pure-proxy)!) on Asset Hub that you use on the Relay Chain. The SS58 encodings are the same; only the chain information (genesis hash, etc.) will change on transaction construction.
```

File: README.md [hunk 5]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
+you use on the Relay Chain. The SS58 encodings are the same; only the chain information (genesis hash, etc.) will change on transaction construction.
```
Comment (IkerAlus) [inline README.md:112]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -106,69 +128,10 @@ In case that an XCM transfer fails to complete successfully, then we will notice
 
 A great resource to learn more about Error Management in XCM is the Polkadot blog post from Dr. Gavin Wood, [XCM Part III: Execution and Error Management](https://www.polkadot.network/blog/xcm-part-three-execution-and-error-management).
 
-### Running a Polkadot Asset Hub node
```
Comment (IkerAlus) [inline README.md:109]: Line 90 of the new version is wrong, we are missing teleport to be mentioned:

Suggestion:

"Currently, foreign assets can be sent and received in the Asset Hub either with a [Reserve Backed Transfer or Teleporting](https://wiki.polkadot.network/docs/learn-xcm-pallet#transfer-reserve-vs-teleport) from other parachains. In both cases, the event emitted when processing the transfer is the `foreignAssets.issued` event. Hence, you should listen to these events, pointing to an address in their system. For this, you must query every new block created, loop through the events array, filter for any `foreignAssets.issued` event, and apply the appropriate business logic."

File: README.md [hunk 7]
```diff
@@ -106,69 +128,10 @@ In case that an XCM transfer fails to complete successfully, then we will notice
...
-
```
Comment (IkerAlus) [inline README.md:110]: Line 97 is wrong, the relay chain is not queried.

Comment (IkerAlus) [reply README.md:110]: please double check that the info of lines 97 and 98 also applies for asset hub when receiving XMC.

File: README.md [hunk 8]
```diff
@@ -106,69 +128,10 @@ In case that an XCM transfer fails to complete successfully, then we will notice
...
-Using the Asset Hub will require running a parachain node to sync the chain. This is very similar to running a Polkadot node, with the addition of some extra flags. You can download the latest release of the [`polkadot-parachain` binary](https://github.com/paritytech/polkadot-sdk/releases/latest/) or build it from [source](https://github.com/paritytech/polkadot-sdk/) with the following commands:
```
Comment (IkerAlus) [inline README.md:111]: Line 105 of new version and other ocurrences: Do not mention system parachains, always talk about Asset Hub

File: README.md [hunk 9]
```diff
@@ -209,7 +172,7 @@ For the case of `foreign-assets`, the sidecar can fetch information associated w
 }
 ```
 
-Using the generic endpoints for the pallets, ssidecar can also retrieve values specific to the configuration of the pallet, such as `constants`, `dispatchables`, `errors`, `events` and `storage`.
+Using the generic endpoints for the pallets, sidecar can also retrieve values specific to the configuration of the pallet, such as `constants`, `dispatchables`, `errors`, `events` and `storage`.
```
Comment (IkerAlus) [inline README.md:205]: Line 139 above and other occurrences: let's call it Sidecar or API Sidecar. Not "the sidecar" etc.

Comment (IkerAlus) [reply README.md:205]: I would remove Line 179 and everything below and add it to the PJS example, Here it is enough if we mention how to submit a transaction (already signed and ready, a byte array). You can reference the relevant example here.

File: README.md [hunk 10]
```diff
@@ -272,6 +235,12 @@ Asset-transfer-api is a library focused on simplifying the construction of asset
 It exposes a reduced set of methods which facilitates users to send transfers to other (para) chains or locally.
 It covers the pallet `assets` and its instances `pool-assets` and `foreign-assets`, as well as local transactions.
 
+#### Tx Wrapper Polkadot
```
Comment (IkerAlus) [inline README.md:222]: Remove this section and any txwrapper mention if there is no foreign assets support on time.

File: README.md [hunk 11]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
+\ --rpc-external
+\ --prometheus-external
+```
+And then run the node with:
+```bash
+$ ./target/release/polkadot-parachain --chain asset-hub-westend
+```
 
-### Foreign Asset Operations
+#### Types of Nodes
```
Comment (IkerAlus) [inline README.md:36]: dont we have any update about the potential different node functioning? like run a archive node for a  parachain with only a full node for relay and so on.

File: README.md [hunk 12]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
 
-The Asset Hub will come with the same tooling suite that Parity Technologies provides for the Relay Chain, namely [API Sidecar](https://github.com/paritytech/substrate-api-sidecar), [TxWrapper Polkadot](https://github.com/paritytech/txwrapper-core/tree/main/packages/txwrapper-polkadot), and the [Asset Transfer API](https://github.com/paritytech/asset-transfer-api). If you have a technical question or issue about how to use one of the integration tools, please file a GitHub issue so a developer can help.
+#### NOS
 
-#### Asset Transfer API
+NoS stands for Node + (API) Sidecar and consists of a simple script to launch a full node of [Asset Hub](https://wiki.polkadot.network/docs/learn-system-chains#asset-hub) together with an instance of [Sidecar](https://github.com/paritytech/substrate-api-sidecar) on a machine (using Docker).
```
Comment (IkerAlus) [inline README.md]: remove link to Asset Hub here.

File: README.md [hunk 13]
```diff
@@ -11,53 +14,72 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
...
 
-`API Sidecar` is a REST service for Relay Chain and parachain nodes. It comes with endpoints to query information about assets and asset balances on the Asset Hub.
-- Asset lookups always use the `AssetId` to refer to an asset class. On-chain metadata is subject to change and thus unsuitable as a canonical index.
-- Please refer to [docs](https://paritytech.github.io/substrate-api-sidecar/dist/) for full usage information. Details on options like how to make a historical query are not included here.
+NoS downloads and maintains archive nodes for at least one Relay Chain and one parachain, in this case Polkadot Relay Chain and the Polkadot Asset Hub respectively, so this will affect the amount of storage needed. At time of writing, the database sizes are 1.7 TiB for the Polkadot Relay Chain node and 133 GiB for the Polkadot Asset Hub node, both running with `pruning=archive`. Sidecar is stateless, so the amount of storage it requires is marginal. You can check the current db size for each type of node [here](https://stakeworld.io/docs/dbsize).
 
-This is the available public instance of sidecar for the Polkadot Asset Hub:
+#### Maintenance
```
Comment (IkerAlus) [inline README.md]: we need to put more datails about monitoring runtime upgrades. As said in my doc, something like [this](https://wiki.polkadot.network/docs/learn-runtime-upgrades#monitoring-runtime-changes) (or just linking it adequately) for Asset Hub

**Review by IkerAlus — CHANGES_REQUESTED** [review]
First review done. I havent checked the examples. 

General structure and level of content is all good but we need to improve a couple of things.

File: README.md (hunk 1 above)
Comment (bee344) [reply README.md:95]: Cool, at first that paragraph was in the guide but I was in doubt on whether we wanted the guide to be more or less constricted to just foreignAssets, so I removed it

**Pull request merged**